### DISCUSSION TITLE
Footnotes support

### DISCUSSION THREAD

**Discussion by adaboese** [mdx-editor/editor#234] [Ideas]
If I type:

```markdown
foo[^1]

[^1]test2
```

even in source mode, it gets converted to:

```markdown
foo\[^1]

\[^1]test2
```

**Comment by petyosi** [comment]
Foot notes are not supported at the moment. They can probably be handled with a plugin, but it's not trivial I believe.

**Reply by Gusis93** [reply]
Do you think it will be added in the future? Its the one thing missing thats keeping me from switching 😞
